src/main/java/uk/gov/hmcts/juror/api/moj/service/PoolCreateServiceImpl.java

```diff
@@ -526,6 +526,8 @@ private JurorPool createJurorPool(String login, String owner, Voters voter,
         jurorPool.setUserEdtq(login);
         jurorPool.setLastUpdate(LocalDateTime.now());
 
+        jurorPool.setResponseEntered(false);
+
```

Comment:
Why do we set this to be false here?

Reply:
We don't this was when I had the response_entered to never be null 😄  but if we move the flag to `juror` this will be redundant anyways

Reply:
updated